tests/integration/features/integration_tests.feature

```diff
@@ -27,15 +27,16 @@ Feature: Integration tests
         When I run a "ccm node1 nodetool flush" command
         When I load 100 rows in the "medusa.test" table
         When I run a "ccm node1 nodetool flush" command
-        When I perform a backup in "full" mode of the node named "first_backup"
+        When I perform a backup in "full" mode of the node named "first_backup" with md5 enabled = "false"
         Then I can see the backup named "first_backup" when I list the backups
         Then I can download the backup named "first_backup" for all tables
         Then I can download the backup named "first_backup" for "medusa.test"
         Then I can see the backup status for "first_backup" when I run the status command
         Then backup named "first_backup" has 16 files in the manifest for the "test" table in keyspace "medusa"
         Then the backup index exists
         Then the backup named "first_backup" has 2 SSTables for the "test" table in keyspace "medusa"
-        Then I can verify the backup named "first_backup" successfully
+        Then I can verify the backup named "first_backup" with md5 enabled = "false" successfully
+        Then I can verify the backup named "first_backup" with md5 enabled = "true" successfully
```

Comment:
Alternatively, I can add 2 different tests if that is more desirable. One for just `I can verify the backup named "first_backup" `, with md5 disabled, and another for `I can verify the backup named "first_backup" with md5 enabled successfully`